Trimble.Modus.Components/Enums/SpinnerColor.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Trimble.Modus.Components.Enums;
+public enum SpinnerColor
+{
+    White,
```

Comment:
Should be `Primary` and `Secondary`

Reply:
Renamed to Primary(Blue),Secondary(White)